ZukoAzula2016/src/org/team1540/zukoazula/PIDTalon.java

```diff
@@ -52,7 +52,7 @@ public PIDTalon(TalonExtendedMotor tem, String name, FloatInput targetSpeed) {
         Cluck.publish(name + " Position", tem.modEncoder().getEncoderPosition());
     }
 
-    private static BooleanInput getThreeState(BooleanInput forceTrue, BooleanInput forceFalse) {
+    public static BooleanInput getThreeState(BooleanInput forceTrue, BooleanInput forceFalse) {
```

Comment:
I find it kind of weird to keep this in PIDTalon. Maybe move it to ZukoAzula, or somewhere else? I'm not sure.